CHANGELOG.md

```diff
@@ -6,20 +6,34 @@
 
 ### Breaking Changes
 
-- Remove the `skip_glotpress` parameter from the `ios_bump_version_release` action [#443]
+_None_
 
 ### New Features
 
-- Add new `buildkite_annotate` action to add/remove annotations from the current build. [#442]
-- Add new `buildkite_metadata` action to set/get metadata from the current build. [#442]
-- Add new `prototype_build_details_comment` action to make it easier to generate the HTML comment about Prototype Builds in PRs. [#449]
+_None_
 
 ### Bug Fixes
 
 _None_
 
 ### Internal Changes
 
+_None_
+
+## 7.0.0
```

Comment:
Out of scope: Interesting that the header the automation generates here do not include links to the `/release/<version>` page.

The template I suggested in the various Apple library repos had that link because I though it was handy. Evidently, it's not a must have feature, otherwise we would have felt the pain of not having it in this library's CHANGELOG.

In fact, the only memories I have of those changelog version links are of _fixing missing or wrongly copy-pasted instances_!

Maybe we'd be better served to omit it elsewhere, too. One seldom needs to open the release page when all the key information is in the CHANGELOG already. Given that, the maintenance savings over time should amply pay off for the extra friction of manually finding a release page every now and then, for example by typing `/release/<version>` in the browser URL bar.

Reply:
Yep same reasoning as you, agree 💯 with pros vs cons. And I think that's probably the reason why I didn't bother add those links in the first place when introducing the toolkit's CHANGELOG… and why I stopped doing them for my personal OSS repos as well